src/site/_data/paths/index.js

```diff
@@ -34,4 +35,5 @@ module.exports = {
   reliable,
   secure,
   vitals,
+  media,
```

Comment:
@jpmedley I moved the collection to the bottom of `/learn` because that's what I've been doing with other collections. If the content ends up being popular then I will bump it up accordingly.